033-OSSDatabaseMigration/Coach/01-discovery.md

```diff
@@ -1,10 +1,10 @@
 # Challenge 1: Assessment 
```

Comment:
Does the title of the challenge need to change as well?  I see the file and challenge named differently on the readme page.

Reply:
Fixed